tests/requirements.txt

```diff
@@ -0,0 +1,8 @@
+pytest
+ragger[speculos]>=1.11.4
+ragger[ledgerwallet]>=1.11.4
+Jinja2==3.1.2
+Flask==2.1.2
```

Comment:
Is there a reason why this Flask version is pinned? This will trigger issues with `Werkzeug` as it latest versions move a function, which [will trigger a bug](https://github.com/LedgerHQ/app-ton-new/actions/runs/6932518093/job/18858084425?pr=6) with this version of Flask.

Speculos [should provide the correct version](https://github.com/LedgerHQ/speculos/blob/master/setup.py#L56) through it dependencies, so either you trust Speculos on this, or you should update this version to `2.3.3` (but this is prone to further dependency issues, as shown here)

Reply:
I took that file from ledger's boilerplate repo, I will modify it to match its latest version now

Reply:
I've fixed that, thank you